config/jobs/kubernetes-sigs/kind/kind-release-blocking.yaml

```diff
@@ -85,11 +82,8 @@ periodics:
       # tell kind CI script to use ipv6
       - name: IP_FAMILY
         value: "ipv6"
-      - name: FOCUS
-        value: "."
-      # TODO(bentheelder): reduce the skip list further
-      - name: SKIP
-        value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Network.should.set.TCP.CLOSE_WAIT.timeout|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|should.provide.basic.identity
+      - name: LABEL_FILTER
+        value: "Feature: isEmpty && !Slow && !Disruptive && !Flaky"
```

Comment:
Slow jobs were skipped before and continue to be skipped.

That begs the question where they should run. I think it would be best to allow running them here instead of setting up a second dedicated job, because:
- less maintenance work for jobs
- easier to see complete test results
- more efficient use of resources when slow tests run in parallel to the normal jobs

The risk is that if we have some really slow tests and they start late, they might not complete in time. Prioritizing slow tests to start early would help, but isn't available yet. As an interim solution we could extend the timeout to be "expected execution time + runtime of slowest job" - might not even be necessary.

Reply:
Yes, absolutely. I just wanted to call out that this PR does *not* change it and start the discussion.